llvm/test/MachineVerifier/convergencectrl/AMDGPU/convergence-readfirstlane64-xfail.mir

```diff
@@ -0,0 +1,46 @@
+# RUN: not --crash llc -mtriple=amdgcn -run-pass=none -verify-machineinstrs -o /dev/null %s 2>&1 | FileCheck %s
```

Comment:
You should not need to introduce any new machine verifier tests, they are not useful. The useful test would be the IR that produces the verifier error, which ideally would be fixed

Reply:
Okay, I'll update with IR's

Reply:
All it needs is one new file in `test/CodeGen/AMDGPU` where 64-bit lane ops are used with a convergence tokens. Mark that as XFAIL. When the issue is fixed, that file can be merged into the existing tests. We don't need to test each of the convergence control intrinsics. It's enough to just have a token on a readlane.

Reply:
I'd still test all 3, but yes an IR test